app/frontend/src/_colors.scss

```diff
@@ -0,0 +1,183 @@
+// Vanilla Colors
```

Comment:
Can you add a comment here, stating which shade is darker, lighter. Like is color-10 darker than color-20?